setup.py

```diff
@@ -14,6 +14,7 @@
 |               |           @ # .`
 '''
 
+

```

Comment:
Remove empty line

Reply:
I referenced the wrong issue in the initial fix, so I had to make a change to fix it.